### PR TITLE
Fix the build on 32-bit architectures

### PR DESCRIPTION
**Pull request by asomers**
Fix the build on 32-bit architectures, and test i686-unknown-freebsd in CI.

### CODE REVIEW

**Review by dlrobertson — APPROVED** [review]
Nice! Thanks for breaking out the lint step!

File: Cargo.toml [hunk 1]
```diff
@@ -19,7 +19,7 @@ targets = [
 ]
```
Comment (dlrobertson) [inline Cargo.toml:19]: Do we need to add `i686-unknown-freebsd`?

Comment (asomers) [reply Cargo.toml:19]: That's just for docs.  Do you think we should build docs for i686-unknown-freebsd too?

Comment (dlrobertson) [reply Cargo.toml:19]: ah right! my bad.

**Pull request merged**